docs/managed_package.rst

```diff
@@ -1,27 +1,155 @@
-Release a managed package
+Release a Managed Package
 =========================
 
+This section outlines how to release a first generation (1GP) Salesforce Managed Package project. Salesforce.org's Release Engineering team practices `CumulusCI Flow<TODO link>`_, which incorporates all of these steps.
```

Comment:
"Managed Package project". When is it capitalized? When is it not?

Reply:
Good question. I think I went a little overboard. I checked current SF docs and looks like it is lower case.

Reply:
Let's not capitalize. As I mentioned, I think I just went overboard with the uppercase writing this up XD